src/api-reference/request/v4.endpoints.request-resources.markdown

```diff
@@ -0,0 +1,755 @@
+  "mainDestination": {
+    "city": "Lyon, FRANCE",
+    "countryCode": "FR",
+    "countrySubDivisionCode": "FR-69",
+    "name": "Lyon, FRANCE"
+  },
+  "policy": {
+    "id": "F4C8BD31CA9D4D6292795BE687EB9B2A"
+  },
+  "travelAgency": {
+    "id": "2EC038D7C3CBBE4ABA0914425064D34F"
+  }
+}
+```
+
+#### HTTP Response
+
+```shell
+200 OK
+```
+
+```json
+{
+  "href": "https://us.api.concursolutions.com/v4/requests/053A479B3C9DD847B02A203C657AE26B",
+  "id": "053A479B3C9DD847B02A203C657AE26B",
+  "approvalStatus": {
+    "code": "NOT_SUBMITTED",
+    "name": "Not Submitted"
+  },
+  "approved": false,
+  "businessPurpose": "Trip to Lyon for company training - Modification of dates and Cost center + Custom Field",
+  "canceledPostApproval": false,
+  "closed": false,
+  "comment": "Company training requires to go to Lyon - Dates and service changed",
+  "creationDate": "2018-05-25T09:17:25.000Z",
+  "custom1": {
+    "value": "Training part of IT Service"
+  },
+  "custom2": {
+    "code": "CEN1",
+    "value": "54F0CBD8833CB348BD45A6C7C621C951",
+    "href": "https://us.api.concursolutions.com/list/v1/items/54F0CBD8833CB348BD45A6C7C621C951"
+  },
+  "custom3": {
+    "code": "CEN1PRO2",
+    "value": "441D6FC50766A044ACC07FF780F1BAD9",
+    "href": "https://us.api.concursolutions.com/list/v1/items/441D6FC50766A044ACC07FF780F1BAD9"
+  },
+  "custom4": {
+    "code": "TRAINING",
+    "value": "3F54AE68BA66EF49A5984E5197202A4D",
+    "href": "https://us.api.concursolutions.com/list/v1/items/3F54AE68BA66EF49A5984E5197202A4D"
+  },
+  "endDate": "2018-07-09",
+  "endTime": "19:00",
+  "everSentBack": false,
+  "expenses": [],
+  "lastModified": "2018-05-25T09:24:34.000Z",
+  "mainDestination": {
+    "countryCode": "FR",
+    "countrySubDivisionCode": "FR-69",
+    "city": "Lyon, FRANCE",
+    "name": "Lyon, FRANCE"
+  },
+  "name": "Company Training - JULY 2018",
+  "owner": {
+    "href": "https://us.api.concursolutions.com/profile/v1/users/c0d9894b-98e2-48d5-86f9-1decde90dd15",
+    "id": "c0d9894b-98e2-48d5-86f9-1decde90dd15",
+    "template": "https://us.api.concursolutions.com/profile/v1/users/{id}"
+  },
+  "pendingApproval": false,
+  "policy": {
+    "id": "F4C8BD31CA9D4D6292795BE687EB9B2A"
+  },
+  "requestId": "333X",
+  "startDate": "2018-07-07",
+  "startTime": "06:15",
+  "totalApprovedAmount": {
+    "value": 0,
+    "currency": "USD"
+  },
+  "totalPostedAmount": {
+    "value": 0,
+    "currency": "USD"
+  },
+  "totalRemainingAmount": {
+    "value": 0,
+    "currency": "USD"
+  },
+  "travelAgency": {
+    "href": "https://us.api.concursolutions.com/v4/travelagencies/2EC038D7C3CBBE4ABA0914425064D34F",
+    "id": "2EC038D7C3CBBE4ABA0914425064D34F",
+    "template": "https://us.api.concursolutions.com/v4/travelagencies/{id}"
+  },
+  "operations": [
+    {
+      "rel": "submit",
+      "href": "https://us.api.concursolutions.com/v4/requests/053A479B3C9DD847B02A203C657AE26B/submit"
+    }
+  ]
+}
+```
+
+### Additional information
```

Comment:
The Additional Information content should be in the main description, under the endpoint heading. We don't use an Additional Information heading. Do the same for all other Additional Information headings.